examples/archive.html

```diff
@@ -11,6 +11,8 @@
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
 
+    <script src="../src/DistortableCollection.js"></script>
```

Comment:
Here, I am not sure we need this or the next line in this PR, maybe these are left over from another pull request? Yes - it looks like you are making a PR from your `main` branch, so this can happen. It's OK as long as you remove these lines now, but in the future you can avoid this by working in a named branch for each new feature, instead of in your main branch!

Reply:
Hii @jywarren 
I’ve made all the required changes and the script I added was for the generateExportJson() so the function could be passed down
And yes yes, next time I’d work on a different branch 
Thank youu

Reply:
Aha, so what will happen is if you compile the code (I believe it's `grunt build`?) It will automatically include all files in the src directory into the files in /dist/, so you don't need to separately do this!

Reply:
Ah, actually it's "npm run build" as shown here: https://github.com/publiclab/Leaflet.DistortableImage/blob/main/CONTRIBUTING.md#setup

Reply:
Ohh
Alright 
I didn’t know that 
Thank youu
Let me make the correction

Reply:
Done 
Thank youu

Reply:
The script tag below it is for the font awesome icon on the button.
The save icon